### ISSUE TITLE
您好，我跑了您的代码，训练的结果好像与理想有出入，请问可以交流一下吗

### ISSUE THREAD

**Issue by guizyu** [WuLynLinux/Lyn#3]
代码的参数没有改，如下：
learning_rate = 0.01
size = 512
delta = 0.03
dL = 0.02
batch_size = 64
batch = 10
c = 3e8
Hz = 0.4e12
最后的结果是这样的，好像并没有收敛：
epoch :1 loss:8.5502 acc:0.9375
epoch :2 loss:3.3796 acc:0.9688
epoch :3 loss:3.6199 acc:0.9688
epoch :4 loss:3.8715 acc:0.9688
epoch :5 loss:4.0078 acc:0.9688
epoch :6 loss:0.0015 acc:1.0000
epoch :7 loss:0.0005 acc:1.0000
epoch :8 loss:0.0000 acc:1.0000
epoch :9 loss:4.0107 acc:0.9688
epoch :10 loss:8.0005 acc:0.9375
epoch :11 loss:4.0004 acc:0.9688
epoch :12 loss:9.9267 acc:0.9219
epoch :13 loss:7.9175 acc:0.9375 
epoch :14 loss:11.9953 acc:0.9062
epoch :15 loss:7.9595 acc:0.9375
epoch :16 loss:2.0000 acc:0.9844
epoch :17 loss:10.0000 acc:0.9219
epoch :18 loss:8.0024 acc:0.9375
epoch :19 loss:5.3605 acc:0.9531 
epoch :20 loss:2.0257 acc:0.9844

**Comment by ghost** [comment]
你好  我们可以讨论交流交流  我最近也在研究这个  由于我的gpu配置不够，所以这个程序我没有跑，我想知道这个程序你运行之后，下载的图片，是关于更新优化后的幅度的图吗，这样就能便于三维建模，3d打印了

**Comment by ghost** [comment]
如果你不介意的话，我们可以在qq或者wechat上进行聊天，便于我们在科研的过程中，相互探讨
